concent_api/core/subtask_helpers.py

```diff
@@ -103,15 +111,18 @@ def _update_timed_out_subtask(subtask: Subtask) -> None:
         )
     elif subtask.state == Subtask.SubtaskState.ADDITIONAL_VERIFICATION.name:  # pylint: disable=no-member
         task_to_compute = deserialize_message(subtask.task_to_compute.data.tobytes())
-        # Worker makes a payment from requestor's deposit just like in the forced acceptance use case.
-        bankster.finalize_payment(
```

Comment:
Could you extract this branch of the condition into a function and use it both here and in the Celery task? In both cases a verification timeout should be processed in exactly the same way. Having it in two separate places makes it very likely that it will get out of sync.

Reply:
But now it will be something different after change from https://github.com/golemfactory/concent/pull/922#discussion_r233454766

Reply:
You could still extract functions like `finalize_claim_for_additional_verification_case()` into a common module and use them in both places.

But well, it does get pretty ugly that way...